scripts/dev_cli.sh

```diff
@@ -423,6 +443,13 @@ cmd_tests() {
     fi
 
     if [ "$integration" = true ]; then
+        # for cvm guest run please do, export GUEST_VM_TYPE=CVM
+        if [ "$GUEST_VM_TYPE" = "CVM" ]; then
```

Comment:
I think the best thing would be to add a new suite of tests (with a new run script) like we do for VFIO or other architectures.

Reply:
We use mod groups in the integration.rs file to differentiate different types of tests. I don't want us to see another axis with the CVM. To avoid duplication of tests you can do refactoring of the tests.

Reply:
I have modified the the framework little bit. Please take another look @rbradford